### PR TITLE
quickemu: 4.9.2 -> 4.9.4

### PR DESCRIPTION
**Pull request by flexiondotorg**
## Description of changes

New upstream release of Quickemu, which changes how some of the internals work which is why the dependencies have changed.

- https://github.com/quickemu-project/quickemu/releases/tag/4.9.3

I am the upstream developer of Quickemu and I've added myself as a maintainer.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>quickemu</li>
    <li>quickgui</li>
  </ul>
</details>

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by FedX-sudo** [comment]
The best practice for packaging has changed since this was packaged, and it is now considered best to package by [name](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md). Refer to this commit for what the new best practices look like to quickemu. https://github.com/NixOS/nixpkgs/pull/310384

**Comment by flexiondotorg** [comment]
@FedX-sudo I've made the changes you recommended and re-ran `nixpkgs-review`

**Review by FedX-sudo — APPROVED** [review]
Tested basic functionality in `/result/bin`. 

EDIT: and used `nixpkgs-review`.

File: pkgs/development/quickemu/default.nix [hunk 1]
```diff
@@ -90,9 +90,9 @@ stdenv.mkDerivation rec {
   passthru.tests = testers.testVersion { package = quickemu; };
 
   meta = with lib; {
```
[the remark this answers is not in the document]
Comment (flexiondotorg) [reply pkgs/development/quickemu/default.nix]: The most recent push resolves this.

File: pkgs/development/quickemu/default.nix [hunk 2]
```diff
@@ -90,9 +90,9 @@ stdenv.mkDerivation rec {
...
-    description = "Quickly create and run optimised Windows, macOS and Linux desktop virtual machines";
+    description = "Quickly create and run optimised Windows, macOS and Linux virtual machines";
     homepage = "https://github.com/quickemu-project/quickemu";
     license = licenses.mit;
```
[the remark this answers is not in the document]
Comment (flexiondotorg) [reply pkgs/development/quickemu/default.nix]: The most recent push resolves this.

File: pkgs/development/quickemu/default.nix [hunk 3]
```diff
@@ -90,9 +90,9 @@ stdenv.mkDerivation rec {
...
-    maintainers = with maintainers; [ fedx-sudo ];
+    maintainers = with maintainers; [ fedx-sudo flexiondotorg ];
```
[the remark this answers is not in the document]
Comment (flexiondotorg) [reply pkgs/development/quickemu/default.nix]: The most recent push resolves this.

**Review by FedX-sudo — APPROVED** [review]

**Comment by a-kenji** [comment]
Result of `nixpkgs-review pr 310409` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>quickemu</li>
    <li>quickgui</li>
  </ul>
</details>

**Review by azahi — APPROVED** [review]

**Comment by me00001** [comment]
`ERROR! You have insufficient RAM to run windows in a VM`
Unable to run windows after this update, Using the older version solves the problem.

**Pull request merged**